src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -169,7 +169,13 @@ private static Checksum GetMetadataChecksumSlow(Solution solution, PortableExecu
             {
                 var serializer = solution.Workspace.Services.GetService<ISerializerService>();
                 var checksum = serializer.CreateChecksum(reference, cancellationToken);
-                return checksum;
+
+                // Include serialization format version in our checksum.  That way if the 
+                // version ever changes, all persisted data won't match the current checksum
+                // we expect, and we'll recompute things.
+                return Checksum.Create(
```

Comment:
can we not create this every single time?

Reply:
We won't... this is in a cache call.  So we only recreate this if reference changes.